### DISCUSSION TITLE
为啥大世界每月开荒老是弹出这个

### DISCUSSION THREAD

**Discussion by Moment0901** [LmeSzinc/AzurLaneAutoScript#1699] [Q&A]
═══════════════════════════════════════════════════════════════════════════════════
                                    OPSIEXPLORE                                    
═══════════════════════════════════════════════════════════════════════════════════
═════════════════════════════════════ OS INIT ═════════════════════════════════════
INFO     17:35:06.348 │ OS INIT                                                    
INFO     17:35:06.376 │ <<< UI ENSURE >>>                                          
INFO     17:35:06.378 │ UI get current page                                        
INFO     17:35:06.388 │ [UI] page_main                                             
INFO     17:35:06.391 │ Goto page_os                                               
INFO     17:35:06.393 │ <<< UI GOTO PAGE_OS >>>                                    
INFO     17:35:06.432 │ Page switch: page_main -> page_campaign_menu               
INFO     17:35:06.435 │ Click (1103,  332) @ MAIN_GOTO_CAMPAIGN                    
INFO     17:35:07.515 │ Page switch: page_campaign_menu -> page_os                 
INFO     17:35:07.517 │ Click ( 690,  419) @ CAMPAIGN_MENU_GOTO_OS                 
INFO     17:35:09.332 │ Page arrive: page_os                                       
INFO     17:35:09.334 │ <<< ZONE INIT >>>                                          
INFO     17:35:09.343 │ Get zone name                                              
INFO     17:35:09.397 │ Loading OCR model: ./bin/cnocr_models/cnocr                
INFO     17:35:09.533 │ [MAP_NAME 0.157s] NA海域东北E                              
INFO     17:35:09.537 │ Map name processed: NA海域东北E                            
INFO     17:35:09.543 │ [Zone] [135|NA Ocean NE Sector E]                          
──────────────────────────────── AFTER AUTO SEARCH ────────────────────────────────
INFO     17:35:09.550 │ AFTER AUTO SEARCH                                          
INFO     17:35:09.576 │ No EMP debuff on current fleet                             
INFO     17:35:09.588 │ [HP]  98%  98%  98%  98%  98%  98%                         
INFO     17:35:09.599 │ [Repair icon] [False, False, False, False, False, False]   
INFO     17:35:09.604 │ No ship found to be below threshold 40%, continue OS       
         exploration                                                               
INFO     17:35:09.608 │ Handle after auto search finished, solved=False            
INFO     17:35:09.614 │ Ash beacon status: gray                                    
INFO     17:35:09.643 │ Loading OCR model: ./bin/cnocr_models/azur_lane            
INFO     17:35:09.733 │ [ASH_COLLECT_STATUS 0.115s] 0/200                          
INFO     17:35:09.779 │ [ASH_DAILY_STATUS 0.042s] 200/200                          
INFO     17:35:09.783 │ Ash beacon fully collected today                           
INFO     17:35:09.786 │ Run auto search, question=True, rescan=False               
───────────────────────────────── OS AUTO SEARCH ──────────────────────────────────
INFO     17:35:09.791 │ OS AUTO SEARCH                                             
INFO     17:35:09.824 │ Click (1253,  542) @ AUTO_SEARCH_OS_MAP_OPTION_OFF         
INFO     17:35:15.244 │ OS auto search finished                                    
INFO     17:35:15.249 │ [HP]  98%  98%  98%  98%  98%  98%                         
INFO     17:35:15.255 │ [Repair icon] [False, False, False, False, False, False]   
──────────────────────────────── AFTER AUTO SEARCH ────────────────────────────────
INFO     17:35:15.259 │ AFTER AUTO SEARCH                                          
INFO     17:35:15.263 │ No EMP debuff on current fleet                             
INFO     17:35:15.269 │ [HP]  98%  98%  98%  98%  98%  98%                         
INFO     17:35:15.274 │ [Repair icon] [False, False, False, False, False, False]   
INFO     17:35:15.277 │ No ship found to be below threshold 40%, continue OS       
         exploration                                                               
INFO     17:35:15.279 │ Handle after auto search finished, solved=False            
───────────────────────────────── CLEAR QUESTION ──────────────────────────────────
INFO     17:35:15.281 │ CLEAR QUESTION                                             
INFO     17:35:15.352 │ No question mark above current fleet on this radar         
──────────────────────────────── AFTER AUTO SEARCH ────────────────────────────────
INFO     17:35:15.355 │ AFTER AUTO SEARCH                                          
INFO     17:35:15.359 │ No EMP debuff on current fleet                             
INFO     17:35:15.365 │ [HP]  98%  98%  98%  98%  98%  98%                         
INFO     17:35:15.370 │ [Repair icon] [False, False, False, False, False, False]   
INFO     17:35:15.373 │ No ship found to be below threshold 40%, continue OS       
         exploration                                                               
INFO     17:35:15.375 │ Handle after auto search finished, solved=False            
═══════════════════════════════════ OS EXPLORE ════════════════════════════════════
INFO     17:35:15.379 │ OS EXPLORE                                                 
INFO     17:35:15.381 │ Last zone: [135|NA Ocean NE Sector E], next zone: [111]    
INFO     17:35:15.384 │ <<< GLOBE GOTO: [111|NA OCEAN E SECTOR A] >>>              
INFO     17:35:15.401 │ Click (1092,  655) @ MAP_GOTO_GLOBE                        
INFO     17:35:18.243 │ Swipe ( 471,  441) -> ( 521,  395)                         
WARNING  17:35:19.453 │ Trying to move in globe, but not in os globe map           
CRITICAL 17:35:19.456 │ Trying to move in globe, but not in os globe map           
CRITICAL 17:35:19.458 │ This is likely to be a mistake of developers, but sometimes
         just random issues                                                        
INFO     17:35:19.461 │ No provider specified, skip sending                        

没报错，但是停止运行了，然后游戏就停在这里
![Screenshot_20221001-185948](https://user-images.githubusercontent.com/78264687/193403997-11c02e67-34c1-4970-9823-1a7116e37f42.png)

**Comment by lemonx001** [comment]
同样经常出这个问题
